pkg/kubectl/cmd/cmd.go

```diff
@@ -235,6 +236,7 @@ Find more information at https://github.com/kubernetes/kubernetes.`,
 	cmds.AddCommand(NewCmdVersion(f, out))
 	cmds.AddCommand(NewCmdExplain(f, out))
 	cmds.AddCommand(NewCmdConvert(f, out))
+	cmds.SetOutput(os.Stdout)
```

Comment:
You'll need to define a custom Help() that outputs to stdout.  We should
not change c.SetOutput() because errors will go to stdout.

On Tue, May 24, 2016 at 9:16 AM, Davanum Srinivas <notifications@github.com>
wrote:

> In pkg/kubectl/cmd/cmd.go
> <https://github.com/kubernetes/kubernetes/pull/26077#discussion_r64387889>
> :
>
> > @@ -235,6 +236,7 @@ Find more information at https://github.com/kubernetes/kubernetes.` <https://github.com/kubernetes/kubernetes.>,
> >  	cmds.AddCommand(NewCmdVersion(f, out))
> >  	cmds.AddCommand(NewCmdExplain(f, out))
> >  	cmds.AddCommand(NewCmdConvert(f, out))
> > +	cmds.SetOutput(os.Stdout)
>
> @smarterclayton <https://github.com/smarterclayton> : Currently "kubectl
> help create" goes to stderr (Please see the bug report). If there's a
> better way, happy to change the code.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/26077/files/abc4e0046bd11952535c7773e37f8c43100051b2#r64387889>
>